src/keylessBackup/KeylessBackupPhoneCodeInput.test.tsx

```diff
@@ -129,4 +131,50 @@ describe('KeylessBackupPhoneCodeInput', () => {
     jest.runOnlyPendingTimers()
     expect(navigate).not.toHaveBeenCalled()
   })
+  it('displays the help bottom sheet and track analytics event when pressing the help button', async () => {
+    const { getByTestId } = renderComponent()
+
+    expect(getByTestId('KeylessBackupPhoneCodeInputHelp')).toBeTruthy()
+    fireEvent.press(getByTestId('KeylessBackupPhoneCodeInputHelp'))
+    expect(getByTestId('KeylessBackupPhoneCodeInput/HelpInfoBottomSheet')).toBeTruthy()
+    expect(ValoraAnalytics.track).toHaveBeenCalledWith(
+      KeylessBackupEvents.cab_phone_verification_help
+    )
+  })
+  it('dismisses help bottom sheet and track analytics event when pressing go back button', async () => {
+    const { getByTestId, queryByTestId } = renderComponent()
+
+    expect(getByTestId('KeylessBackupPhoneCodeInputHelp')).toBeTruthy()
+    fireEvent.press(getByTestId('KeylessBackupPhoneCodeInputHelp'))
+    expect(getByTestId('KeylessBackupPhoneCodeInput/HelpInfoBottomSheet/GoBack')).toBeTruthy()
+    fireEvent.press(getByTestId('KeylessBackupPhoneCodeInput/HelpInfoBottomSheet/GoBack'))
+    expect(queryByTestId('KeylessBackupPhoneCodeInput/HelpInfoBottomSheet')).toBeFalsy()
```

Comment:
This is failing in a unit test, not sure how else to test that the `Go back` button dismisses the bottom sheet. In manual testing it correctly dismisses it.

Reply:
I'm pretty sure this is because the bottom sheet component is technically always rendered, just not necessarily visible? e.g., if you add `expect(getByTestId('KeylessBackupPhoneCodeInput/HelpInfoBottomSheet')).toBeTruthy()` to the beginning of this test, it won't fail early -- trying to see if there's a way to detect whether or not the bottom sheet is actually _visible_ or not.